rotkehlchen/db/filtering.py

```diff
@@ -441,10 +441,10 @@ def prepare(self) -> Tuple[List[str], List[Any]]:
 
 
 @dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
-class DBMultiStringFilter(DBFilter):
+class DBMultiValueFilter(DBFilter):
```

Comment:
Wouldn't it be better to have a `DBMultiValueFilter` with a generic Type[T] for the list of values? And then create the DBMultistringFilter as Type being String and the bytes one etc. ?

Otherwise typing won't work.